contracts/ownership/Whitelist.sol

```diff
@@ -28,12 +29,27 @@ contract Whitelist is Ownable {
    * @param addr address
    * @return true if the address was added to the whitelist, false if the address was already in the whitelist
    */
-  function addAddressToWhitelist(address addr) onlyOwner public returns(bool success) {
-    if (!whitelist[addr]) {
-      whitelist[addr] = true;
-      emit WhitelistedAddressAdded(addr);
-      success = true;
+  function addAddressToWhitelist(address addr)
+    onlyOwner
+    public
+  {
+    // silent fail if address already exists (backwards compat)
```

Comment:
The failure wasn't entirely silent because there was a boolean return value that was `false` for failures.

Since we removed the return value it is now actually silent, and this makes it not really backwards compatible. That said, I can't make up my mind on what would be the best behavior.

Reply:
True. I've come around on boolean returns for now, at least until status codes are a thing. That said, I don't think this method really benefits from a boolean return, so I'm just going to remove this backwards compatibility attempt